### DISCUSSION TITLE
23 年某项目成果缺少明确的 License 申明

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [summer-ospp/help#235] [Show and tell]
如下图所示

![image](https://github.com/summer-ospp/help/assets/1450685/64bbb13e-89f9-45b4-9b64-212a1c8e7aa6)
![image](https://github.com/summer-ospp/help/assets/1450685/5834e130-940e-44dd-bcad-9d307e0c4894)

## 相关链接
* [将codefever应用发布到渠成应用市场](https://summer-ospp.ac.cn/2023/org/prodetail/23aed0025?list=org&navpage=org)

**Comment by LinuxSuRen** [comment]
类似的情况还包括 https://summer-ospp.ac.cn/2023/org/prodetail/2341f0589?list=org&navpage=org

![image](https://github.com/summer-ospp/help/assets/1450685/1519cdbc-c320-426c-9afc-98653538f1b4)
![image](https://github.com/summer-ospp/help/assets/1450685/89fddd93-8848-4a96-8c5f-323ccb9237b5)
![image](https://github.com/summer-ospp/help/assets/1450685/2084b687-ac2e-46cd-9290-29be6fb018d5)

这个案例既不支持 Fork 也没用 License 文件。

**Comment by LinuxSuRen** [comment]
这个案例的产出居然只是一个 ReadMe 文件，如果我理解错误，请帮忙指出。

![image](https://github.com/summer-ospp/help/assets/1450685/1ea05b90-7fee-4c59-be01-a92c7e3db3f2)
![image](https://github.com/summer-ospp/help/assets/1450685/2580c43a-6e7a-4d48-99a0-ce337f1c0bf5)

## 相关链接
* https://github.com/Tongsuo-Project/data-privacy-solutions
* https://summer-ospp.ac.cn/2023/org/prodetail/23e4d0484?lang=zh&list=pro

**Reply by brsvh** [reply]
> 

你找错位置了，_查找项目_ 似乎只是在项目寻找学生的时候适用。

![image](https://github.com/summer-ospp/help/assets/63050399/bdda11f8-697d-4aa3-8965-fedaa01d7497)

你可以查看这里的贡献是否真实存在，我没有细看

**Reply by brsvh** [reply]
不过在展示项目之初填写了与最终预期成果不同的仓库确实显的不够专业和奇怪了。

**Reply by LinuxSuRen** [reply]
不对呀，你这个截图的是王同学，但我给的链接里申请中选的是贾同学。（我又仔细看了下，这个社区有两个项目，分别由前面两位同学中选。我猜测贾同学没有顺利完成项目。）

**Reply by LinuxSuRen** [reply]
不过，根据你的提示，我感觉如果申请、结果公示的页面之间如果有链接跳转的话，就更方便了。

**Reply by brsvh** [reply]
+1

**Reply by HungryFish034** [reply]
这个情况一般是社区在进程中会更改为正确的项目成果仓库，但组委会并不鼓励这种行为，在 24 年的活动中，组委会在审核项目发布阶段也会将这条规则重点关注。

另外，项目详情页也会在中选信息的基础上增加结项信息。
